contracts/ContinuousFundraising.sol

```diff
@@ -87,24 +101,27 @@ contract ContinuousFundraising is ERC2771Context, Ownable2Step, Pausable, Reentr
      * @param _currency currency used to pay for the token mint. Must be ERC20, so ether can only be used as wrapped ether (WETH)
      * @param _token token to be sold
      */
-    constructor(
-        address _trustedForwarder,
+    function initialize(
+        address _owner,
         address _currencyReceiver,
         uint256 _minAmountPerBuyer,
         uint256 _maxAmountPerBuyer,
         uint256 _tokenPrice,
         uint256 _maxAmountOfTokenToBeSold,
         IERC20 _currency,
         Token _token
-    ) ERC2771Context(_trustedForwarder) {
+    ) external initializer {
+        require(_owner != address(0), "owner can not be zero address");
+        __Ownable2Step_init(); // sets msgSender() as owner
+        _transferOwnership(_owner); // sets owner as owner
```

Comment:
why is this neccesary?

Reply:
This is necessary because we use our platform hot wallet to call the clone factory -> after `__Ownable2Step_init();`, the platform hot wallet will be owner. With `_transferOwnership(_owner);`, we make the founder owner.